app/views/cases/shared/_request_method.html.slim

```diff
@@ -0,0 +1,3 @@
+tr.request-method
+  th = t('common.case.request_method')
+  td = case_details.request_method.humanize
```

Comment:
It's best not to use the `#humanize` method, as that's not doing proper i18n, can give you unexpected results and also rely on a bad practice of coupling internal identifiers to external "human-readable" strings.

Better to do explicit translation with locales, for example:

```ruby
t(case_details.request_method, scope: 'common.case.request_method.choices')
```

I know many other places in the codebase are using `#humanize` unfortunately. Something for a future refactor...

Reply:
Thanks, it did occur to me to wonder why we weren't using the locale for the enum value display as well but couldn't think of the solution. That's great. I think the change is outside the scope of this PR as there's a bunch of these partials that all use this approach, though